phpcs.xml

```diff
@@ -45,165 +44,89 @@
 
 		<!-- Remove WP rules in Bulk -->
+		<exclude name="WordPress.Arrays"/>
 		<exclude name="WordPress.Files.FileName"/>
 		<exclude name="WordPress.NamingConventions"/>
 		<exclude name="WordPress.WhiteSpace"/>
 
-		<!-- These rules are already excluded with above bulk rule-exclusions
-		<exclude name="WordPress.Files.FileName.NotHyphenatedLowercase"/>
-		<exclude name="WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase"/>
-		<exclude name="WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase"/>
-		<exclude name="WordPress.WhiteSpace.ControlStructureSpacing"/>
-		<exclude name="WordPress.WhiteSpace.DisallowInlineTabs.NonIndentTabsUsed"/>
-		<exclude name="WordPress.WhiteSpace.OperatorSpacing.NoSpaceAfter"/>
-		<exclude name="WordPress.WhiteSpace.OperatorSpacing.NoSpaceBefore"/>
-		<exclude name="WordPress.WhiteSpace.PrecisionAlignment.Found"/>
-		-->
-
 		<!-- Remove WP Specific rules. -->
-		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound"/>
-		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.ArrayItemNoNewLine"/>
-		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.NoSpaceAfterArrayOpener"/>
-		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.NoSpaceBeforeArrayCloser"/>
-		<exclude name="WordPress.Arrays.ArrayDeclarationSpacing.SpaceAfterKeyword"/>
-		<exclude name="WordPress.Arrays.ArrayIndentation.ItemNotAligned"/>
-		<exclude name="WordPress.Arrays.ArrayIndentation.CloseBraceNotAligned"/>
-		<exclude name="WordPress.Arrays.ArrayKeySpacingRestrictions.NoSpacesAroundArrayKeys"/>
-		<exclude name="WordPress.Arrays.ArrayKeySpacingRestrictions.SpacesAroundArrayKeys"/>
-		<exclude name="WordPress.Arrays.CommaAfterArrayItem.CommaAfterLast"/>
-		<exclude name="WordPress.Arrays.CommaAfterArrayItem.NoComma"/>
-		<exclude name="WordPress.Arrays.CommaAfterArrayItem.NoSpaceAfterComma"/>
-		<exclude name="WordPress.Arrays.CommaAfterArrayItem.SpaceAfterComma"/>
-		<exclude name="WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned"/>
 		<exclude name="WordPress.Classes.ClassInstantiation.MissingParenthesis"/>
 		<exclude name="WordPress.CodeAnalysis.AssignmentInCondition.Found"/>
-		<exclude name="WordPress.CodeAnalysis.AssignmentInCondition.FoundInWhileCondition"/>
-		<exclude name="WordPress.CodeAnalysis.EmptyStatement.SemicolonWithoutCodeDetected"/>
 		<exclude name="WordPress.DB.DirectDatabaseQuery.DirectQuery"/>
 		<exclude name="WordPress.DB.DirectDatabaseQuery.NoCaching"/>
-		<exclude name="WordPress.DB.SlowDBQuery.slow_db_query_meta_key"/>
-		<exclude name="WordPress.DB.SlowDBQuery.slow_db_query_meta_value"/>
+		<exclude name="WordPress.PHP.DisallowShortTernary.Found"/>
 		<exclude name="WordPress.PHP.StrictComparisons.LooseComparison"/>
 		<exclude name="WordPress.PHP.StrictInArray.MissingTrueStrict"/>
 		<exclude name="WordPress.PHP.YodaConditions.NotYoda"/>
+		<exclude name="WordPress.Security.ValidatedSanitizedInput.InputNotValidated"/>
```

Comment:
It seems like sanitized input is one of the most important sniffs. Why exclude it?

Reply:
Good catch!